src/index.tsx

```diff
@@ -1,14 +1,15 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import "./i18n";
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
```

Comment:
przydał by się jakiś prettier config, żeby za chwilę nie było pełno takich zmian

Reply:
A jaka reguła prettiera jest za takie coś odpowiedzialna? Nie rozumiem też na czym ta reguła miałaby polegać?

Reply:
https://prettier.io/docs/en/options.html

Reply:
skopiuj prettierconfig z poprzedniego projektu

Reply:
To już zrobiłam :)